src/apps/ENGINE/file_service.cpp

```diff
@@ -129,7 +129,14 @@ std::vector<std::filesystem::path> FILE_SERVICE::_GetFsPathsByMask(const char *s
     }
 
     std::filesystem::path curPath;
-    for (auto &dirEntry : std::filesystem::directory_iterator(srcPath))
+    std::error_code ec;
+    auto it = std::filesystem::directory_iterator(srcPath, ec);
+    if (ec)
+    {
+        return result;
```

Comment:
Should we also print an error in the log? `core.tracelog->error()`

Reply:
I think it should be a warning (not an error). @vorot93 Could you please add logging before return?